app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  def update_comments_counter
+    update(comments_count: comments.count)
+  end
```

Comment:
- [ ] @shuhad786 it would have been nice if you ensured your post count is stored in the CommentCounter, this will enable an easier display of comments counts in your views file in future milestones.